doc/concepts/configuration/configuration_memtx.rst

```diff
@@ -0,0 +1,233 @@
+..  _configuration_memtx:
+
+In-memory storage
+=================
+
+**Example on GitHub**: `memtx <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/memtx>`_
+
+In Tarantool, all data is stored in random-access memory (RAM) by default.
+For this purpose, the :ref:`memtx <engines-memtx>` storage engine is used.
+
+This topic describes how to define settings related to in-memory storage in the
+:ref:`memtx <configuration_reference_memtx>` section of a :ref:`YAML configuration <configuration>`.
+
+..  NOTE::
+
+    To calculate the required amount of memory, you can use the
+    `sizing calculator <https://www.tarantool.io/en/sizing_calculator/>`_.
+
+..  _configuration_memtx-memory:
+
+Configure the memory size
+-------------------------
+
+In Tarantool, data is stored in spaces.
+Each space consists of tuples -- the database records.
+To specify the amount of memory that Tarantool allocates to actually store tuples, use the
+:ref:`memtx.memory <configuration_reference_snapshot_dir>` configuration option.
```

Comment:
Looks like `configuration_reference_snapshot_dir` is a wrong link. Please check all other links to reference.